glean-core/android/src/main/java/mozilla/telemetry/glean/debug/GleanDebugActivity.kt

```diff
@@ -123,15 +136,25 @@ class GleanDebugActivity : Activity() {
 
         // This Activity can be used to tag tests on CI or start products with specific
         // options. We need to make sure to retain and propagate all the options that
-        // we were passed to the next intent. Our strategy:
-        // - use the `Intent` copy constructor to copy all the intent options from the
+        // we were passed to the next intent. Our 3 steps strategy:
+        // 1. use the `Intent` copy constructor to copy all the intent options from the
         //   intent which started this activity;
-        // - get the main launch intent for the product using the Glean SDK;
-        // - change the starting "component" and package to the one from the previous step.
         val nextIntent = Intent(intent)
-        val defaultLaunchIntent = packageManager.getLaunchIntentForPackage(packageName)!!
-        nextIntent.component = defaultLaunchIntent.component
-        nextIntent.`package` = defaultLaunchIntent.`package`
+
+        // 2. get the main launch intent for the product using the Glean SDK or lookup
+        //    the one passed as a command line argument to this activity.
+        val launchIntent = packageManager.getLaunchIntentForPackage(packageName)!!
+        nextIntent.`package` = launchIntent.`package`
```

Comment:
Is that special syntax for otherwise-reserved keywords in the language?

Reply:
Yup, it is :)